bundles/org.openhab.binding.plugwiseha/src/main/resources/ESH-INF/i18n/plugwiseha_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
+# FIXME: please do not add the file to the repo if you add or change no content
+# binding
+binding.plugwiseha.name = <Your localized Binding name>
```

Comment:
Remove this file

Reply:
Thanks for your help! It's my first time contributing so I expect some 'hurdles' along the way ;-)